perun-cli/Perun/beans/FormItem.pm

```diff
@@ -0,0 +1,294 @@
+        $ordnum = 0;
+    }
+
+    my $forDelete;
+    if (defined($self->{_forDelete})) {
+        $forDelete = $self->{_forDelete};
+    } else {
+        $forDelete = undef;
+    }
+
+    my $i18n;
+    if (defined($self->{_i18n})) {
+        $i18n = $self->{_i18n};
+    } else {
+        $i18n = undef;
+    }
+
+    return { id                   => $id,
+        shortname                 => $shortname,
+        required                  => $required,
+        type                      => $type,
+        federationAttribute       => $federationAttribute,
+        perunSourceAttribute      => $perunSourceAttribute,
+        perunDestinationAttribute => $perunDestinationAttribute,
+        regex                     => $regex,
+        applicationTypes          => $applicationTypes,
+        ordnum                    => $ordnum,
+        forDelete                 => $forDelete,
+        i18n                      => $i18n,
+        beanName                  => 'FormItem'
+    };
+}
+
+sub getId
+{
+    my $self = shift;
+
+    return $self->{_id};
+}
+
+sub getShortname
+{
+    my $self = shift;
+
+    return $self->{_shortname};
+}
+
+sub setShortname
+{
+    my $self = shift;
+    $self->{_shortname} = shift;
+
+    return;
+}
+
+sub getRequired
+{
+    my $self = shift;
+
+    return $self->{_required};
+}
+
+sub setRequired
+{
+    my $self = shift;
+    $self->{_required} = shift;
+
+    return;
+}
+
+sub getType
+{
+    my $self = shift;
+
+    return $self->{_type};
+}
+
+sub setType
+{
+    my $self = shift;
+    $self->{_type} = shift;
+
+    return;
+}
+
+sub getFederationAttribute
+{
+    my $self = shift;
+
+    return $self->{_federationAttribute};
+}
+
+sub setFederationAttribute
+{
+    my $self = shift;
+    $self->{_federationAttribute} = shift;
+
+    return;
+}
+
+sub getPerunSourceAttribute
+{
+    my $self = shift;
+
+    return $self->{_perunSourceAttribute};
+}
+
+sub setPerunSourceAttribute
+{
+    my $self = shift;
+    $self->{_perunSourceAttribute} = shift;
+
+    return;
+}
+
+sub getPerunDestinationAttribute
+{
+    my $self = shift;
+
+    return $self->{_perunDestinationAttribute};
+}
+
+sub setPerunDestinationAttribute
+{
+    my $self = shift;
+    $self->{_perunDestinationAttribute} = shift;
+
+    return;
+}
+
+sub getRegex
+{
+    my $self = shift;
+
+    return $self->{_regex};
+}
+
+sub setRegex
+{
+    my $self = shift;
+    $self->{_regex} = shift;
+
+    return;
+}
+
+sub getApplicationTypes
+{
+    my $self = shift;
+
+    return $self->{_applicationTypes};
+}
+
+sub setApplicationTypes
+{
+    my $self = shift;
+    $self->{_applicationTypes} = shift;
+
+    return;
+}
+
+sub getOrdnum
+{
+    my $self = shift;
+
+    return $self->{_ordnum};
+}
+
+sub setOrdnum
+{
+    my $self = shift;
+    $self->{_ordnum} = shift;
+
+    return;
+}
+
+sub getForDelete
+{
+    my $self = shift;
+
+    return $self->{_forDelete};
+}
+
+sub setForDelete
+{
+    my $self = shift;
+    $self->{_forDelete} = shift;
```

Comment:
``forDelete`` is boolean property. See, how we handle getter/setter in ``User.pm`` for ``serviceUser`` property.